pkgs/by-name/ex/extractpdfmark/package.nix

```diff
@@ -0,0 +1,39 @@
+{ autoconf
+, automake
+, fetchFromGitHub
+, ghostscript
+, lib
+, pkg-config
+, poppler
+, stdenv
+, texlive
+}:
+
+stdenv.mkDerivation rec {
+  pname = "extractpdfmark";
+  version = "1.1.1";
+
+  src = fetchFromGitHub {
+    owner = "trueroad";
+    repo = "extractpdfmark";
+    rev = "v${version}";
+    hash = "sha256-pNc/SWAtQWMbB2+lIQkJdBYSZ97iJXK71mS59qQa7Hs=";
+  };
+
+  nativeBuildInputs = [ autoconf automake pkg-config ];
+  buildInputs = [ ghostscript poppler texlive.combined.scheme-minimal ];
+
+  preConfigure = ''
+    sh ./autogen.sh
```

Comment:
What about `autoreconfHook`?

Reply:
Indeed, I can use it if I create an empty `config.rpath`. Done.